token-metadata/program/src/instruction/mod.rs

```diff
@@ -297,6 +315,7 @@ pub enum MetadataInstruction {
     #[account(8, name="token_program", desc="Token program")]
     #[account(9, name="system_program", desc="System program")]
     #[account(10, optional, name="rent", desc="Rent info")]
+    #[account(11, name="sysvar_instructions", desc="Instructions sysvar account")]
```

Comment:
Does the `metadata` account need to be writable?

Reply:
Yes, thanks!